api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -340,6 +340,13 @@ type AgentClusterInstallList struct {
 	Items           []AgentClusterInstall `json:"items"`
 }
 
+type CaCertificateReference struct {
+	// Namespace is the namespace of the secret containing the CA certificate.
```

Comment:
Add that the certificate should be base64 encoded

Reply:
Fixed